cqlDB/seed.js

```diff
@@ -0,0 +1,89 @@
+const fs = require('fs');
+const faker = require('faker');
+
+// use mongodb for generating script
+const db = require('./index.js');
+const Product = require('./model.js');
+
+var filepath = "data.txt";
+// flag 'a': Open file for appending. The file is created if it does not exist.
+var stream = fs.createWriteStream(filepath, {flags: 'a'});
+
+var generateEntry = function(idCount) {
+
+  var randomTitle = faker.commerce.productName();
+  var randomDescription = faker.lorem.paragraph();
+  var randomRating = Math.random() * 5;
+  var variations = [];
+  var imageCount = 1;
+  var numOfVariations = Math.ceil((Math.random() * 4));
+
+  for (var i = 0; i < numOfVariations; i++) {
+    var randomColor = faker.commerce.color(); // Fender Stratocaster
+    var randomCost = faker.commerce.price(); // 492.00
+    var numOfImages = Math.ceil((Math.random() * 5));
+    var randomImages = [] // 4 random image urls
+
+    var createImage = function() {
+      var url = 'http://picsum.photos/seed/'
+      var urlend='/846/1038'
```

Comment:
A couple missing semicolons? May need to check linter config (or run it if you'd forgotten).